src/data/equiv/algebra.lean

```diff
@@ -505,12 +510,26 @@ variables [semiring α] [semiring β]
 def to_ring_hom (e : α ≃+* β) : α →+* β :=
 { .. e.to_mul_equiv.to_monoid_hom, .. e.to_add_equiv.to_add_monoid_hom }
 
+/-- Reinterpret a ring equivalence as a monoid homomorphism. -/
+def to_monoid_hom (e : α ≃+* β) : α →* β := e.to_ring_hom.to_monoid_hom
```

Comment:
I'm not sure this is a good idea; it saves a few keystrokes, but will surely require more lemmas to be written...

Reply:
@semorrison Would an `abbreviation` help here?

Reply:
Okay.